### PR TITLE
FORMS-9265 Include editable template for fragment 2.0 as part of archetype.

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/.content.xml [hunk 1]
```diff
@@ -8,4 +8,7 @@
     #if ( $includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y" )
     <af-page-v2/>
     #end
+    #if ( $aemVersion == "cloud" and ( $includeForms == "y" or $includeFormsenrollment == "y" or $includeFormscommunications == "y" or $includeFormsheadless == "y") )
+    <afv2-fragment-page/>
+    #end
```
Comment (pankaj-parashar) [inline src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/.content.xml:13]: Is fragment 2.0 applicable to the cloud only?

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/thumbnail.png
Comment (pankaj-parashar) [inline src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/thumbnail.png:1]: We can have fragment specific thumbnail if possible.

**Review by pankaj-parashar — COMMENTED** [review]
LGTM. some minor queries.

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/thumbnail.png
Comment (sakshi-arora1) [reply src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/thumbnail.png:1]: Yes I have raised request with Utkarsh gill.

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/.content.xml (hunk 1 above)
Comment (sakshi-arora1) [reply src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/.content.xml:13]: Currently, till it gets sideported to release/650. I have raised PR for that as well in core components, once its merged, this check will go away.

**Review by pankaj-parashar — APPROVED** [review]

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/initial/.content.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        jcr:primaryType="cq:PageContent"
+        sling:resourceType="${appId}/components/adaptiveForm/page"
+#if ( $aemVersion == "cloud")
+        sling:configRef="/conf/${appId}/forms"
+#end
+        guideComponentType="fd/af/templates">
+        <guideContainer
+            fd:version="2.1"
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="${appId}/components/adaptiveForm/fragmentcontainer"
```
Comment (barshat7) [inline src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/initial/.content.xml:15]: Is this proxy component present in archetype?

Comment (sakshi-arora1) [reply src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/initial/.content.xml:15]: no, one will need to add it, as this is behind feature toggle. But this is a good point, template type will only function if proxy components are present. @rismehta  Do we add proxy components for components which are behind feature toggles?

Comment (sakshi-arora1) [reply src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/template-types/afv2-fragment-page/initial/.content.xml:15]: Added the proxy components as well.

**Pull request closed without merging**